src/Services/Identity/Identity.API/Identity.API.csproj

```diff
@@ -1,9 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
+
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <RootNamespace>OpenCodeFoundation.eSchool.Identity.API</RootNamespace>

```

Comment:
Please keep the `RootNamespace` block. It helps `Visual Studio` to create new class/interface by appending the namespace.